egrismore/Lab4.py

```diff
@@ -0,0 +1,29 @@
+__author__ = 'emilygrismore'
+"""checks requirements for and runs the magic 1089"""
+
+
+def flip(i):
+    """reverses the number"""
+    int(str(i)[::-1])
+    return i
+
+
+def decrease(i):
+    """checks to see if the #'s are in descending order"""
+    largest = 10
+    for new in str(i):
+        number = int(new)
+        if number >= largest:
+            return False
+        else:
+            largest = number
+    return True
+
+
+def trick():
```

Comment:
can you separate out the trick from the testing of the trick?

Move the for loop and the decrease() check into the unit test and then just perform the trick in this function.